open-metadata-implementation/access-services/data-platform/data-platform-spring/src/main/java/org/odpi/openmetadata/accessservices/dataplatform/server/spring/DataPlatformOMASResource.java

```diff
@@ -45,11 +46,19 @@ public GUIDResponse createSoftwareServerCapability(@PathVariable("serverName") S
      * @param qualifiedName the qualified name
      * @return the software server capability by qualified name
      */
-    @GetMapping(path = "/software-server-capabilities/{qualifiedName}")
-    public RegistrationRequestBody getSoftwareServerCapabilityByQualifiedName(@PathVariable String serverName,
-                                                                              @PathVariable String userId,
-                                                                              @PathVariable String qualifiedName) {
+    @GetMapping(path = "/software-server-capability/{qualifiedName}")
+    public DataPlatformRegistrationRequestBody getSoftwareServerCapabilityByQualifiedName(@PathVariable String serverName,
+                                                                                          @PathVariable String userId,
+                                                                                          @PathVariable String qualifiedName) {
         return restAPI.getSoftwareServerCapabilityByQualifiedName(serverName, userId, qualifiedName);
     }
 
+    @PostMapping(path = "/deployed-database-schema")
+    public GUIDResponse createDeployedDatabaseSchema(@PathVariable("serverName") String serverName,
+                                                       @PathVariable("userId") String userId,
+                                                       @RequestBody DeployedDatabaseSchemaRequestBody requestBody) {
+        return restAPI.createDeployedDatabaseSchema(serverName, userId, requestBody);
```

Comment:
This endpoint should return error response in case of error, so signature should change to return a more generic response type to be .able to cover error response as well. This can be done in a separate PR.